src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -4,23 +4,37 @@
 #nullable disable
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
+using Microsoft.AspNetCore.Razor.LanguageServer.Test.Common;
+using Microsoft.AspNetCore.Razor.Test.Common;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Razor.Workspaces.Extensions;
 using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 using Microsoft.VisualStudio.LanguageServerClient.Razor.Extensions;
+using Microsoft.VisualStudio.LanguageServerClient.Razor.Test;
 using Microsoft.VisualStudio.Test;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Operations;
 using Microsoft.VisualStudio.Threading;
+using Microsoft.VisualStudio.Utilities;
 using Moq;
 using Xunit;
+using CompletionContext = Microsoft.VisualStudio.LanguageServer.Protocol.CompletionContext;
+using CompletionItem = Microsoft.VisualStudio.LanguageServer.Protocol.CompletionItem;
+using CompletionList = Microsoft.VisualStudio.LanguageServer.Protocol.CompletionList;
+using CompletionOptions = Microsoft.VisualStudio.LanguageServer.Protocol.CompletionOptions;
+using CompletionTriggerKind = Microsoft.VisualStudio.LanguageServer.Protocol.CompletionTriggerKind;
 using Range = Microsoft.VisualStudio.LanguageServer.Protocol.Range;
 
 namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
 {
+    [UseExportProvider]
     public class CompletionHandlerTest : HandlerTestBase
```

Comment:
Some of the tests in this class were redundant or no longer applicable and were removed (probably ~2-3 tests in total). For example, there was a test that tested per-item TextEdits, but since C# uses ItemDefaults now, the test no longer seemed necessary.